openfe/utils/system_probe.py

```diff
@@ -499,29 +516,34 @@ def log_system_probe(level=logging.DEBUG,
     hostname = logging.getLogger(basename + ".hostname")
     loggers = [base, gpu, hostname]
     if any(l.isEnabledFor(level) for l in loggers):
-        sysinfo = _probe_system(pl_paths)['system information']
+        sysinfo = _probe_system(pl_paths)["system information"]
         base.log(level, "SYSTEM CONFIG DETAILS:")
         hostname.log(level, f"hostname: '{sysinfo['hostname']}'")
-        if gpuinfo := sysinfo['gpu information']:
+        if gpuinfo := sysinfo["gpu information"]:
```

Comment:
Long term, it may be better to just not look for a GPU if we don't ask for one.

Reply:
This might be a gufe-level discussion. There's a proposal in gufe to expand the `Context` object to include "what kinds of resources might I be sitting on", this would be a good place to tie this into?

Reply:
Yes, the system probe stuff could have a home in GUFE since it really isn't opinionated to openfe